crates/biome_css_semantic/src/semantic_model/builder.rs

```diff
@@ -173,3 +185,23 @@ impl SemanticModelBuilder {
         }
     }
 }
+
+fn resolve_selector(current: &str, parent: &[Selector]) -> Vec<String> {
```

Comment:
```suggestion
fn resolve_selector(current: &str, parents: &[Selector]) -> Vec<String> {
```

nit: `parents`  is more accurate since it's a slice of selectors?